### PR TITLE
[DOCS] Document missing data types for node stats API's response parameters

### LINKED ISSUE #22199: [DOCS] Document more return values of Nodes Stats, Cluster State and Cluster Stats
We document some Stat APIs return values in full, and some not. Some of the undocumented values are pretty specific to Elasticsearch, and therefore could benefit users a lot if they were explained.

The following groups of [Nodes Stats](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-nodes-stats.html) are not documented:
- http
- breaker
- jvm
- thread_pool
- transport
- discovery

The [Cluster State API](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-state.html) documentation does not list details of any of its groups:
- version
- master_node
- nodes
- routing_table
- metadata
- blocks

The [Cluster Stats API](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-stats.html) documentation also does not document its return values.

### LINKED ISSUE #47524: [DOCS] Document response body sections for node and cluster stats APIs
Node stats API:
- [x] indices #47525
- [x] jvm #49500
- [x] thread_pool #50330
- [x] transport #50473
- [x] http #50473
- [x] breakers #50509
- [x] script #50509
- [x] discovery #50509
- [x] collapse and reorder response stats #51063
- [ ] Add missing datatypes #53475

Cluster stats API:

- [x] header metadata #50527
- [x] indices  #50527
- [x] nodes #52813

### PR DESCRIPTION
**Pull request by jrodewig**
Documents missing data types for several response parameters returned
by the node stats API.

Also adds several missing human-readable parameters returned by the API.

Closes #22199. Closes #47524.

### CODE REVIEW

File: docs/reference/cluster/nodes-stats.asciidoc [hunk 1]
```diff
@@ -1316,9 +1469,13 @@ Number of compatible differences between published cluster states.
     (integer)
     Number of documents transformed by the processor.
 
+`ingest.pipelines.<pipeline_id>.<processor>.time`::
```
Comment (jbaiera) [inline docs/reference/cluster/nodes-stats.asciidoc]: This should be `ingest.pipelines.<pipeline_id>.processors.<processor>.time` since the processor object is nested under an array field. This should probably be updated for all of the processor specific stats in this reference, though that might be fine to do in a separate PR.

**Review by jbaiera — APPROVED** [review]
This LGTM. The suggestion could be in this PR or a separate one. Sorry it took so long. I like to be thorough on these 😃

File: docs/reference/cluster/nodes-stats.asciidoc (hunk 1 above)
Comment (jrodewig) [reply docs/reference/cluster/nodes-stats.asciidoc]: Good catch. Fixed this up with 28f5be11954e88bc19baf66387ab0066a8b22459.

**Comment by jrodewig** [comment]
Thanks so much @jbaiera. I appreciate your thoroughness!

**Comment by jrodewig** [comment]
### Backport commits
master 52e8f6d 
7.x  2fdf6b2
7.6 ed3a4ef
